Polyhedron/demo/Polyhedron/resources/solid_wireframe_shader.geom

```diff
@@ -27,7 +27,7 @@ void main(void)
   //linearized arbitrary Z offset to keep the widelines in front of the edges as much as possible 
   //Problem if coef is not 0, edges might be visible through faces.
   float coef = 0.0f;
-  float z_offset = (2.0 * near) / (far + near - coef * (far - near)) - (2.0 * near) / (far + near);
+  float z_offset = 0;//(2.0 * near) / (far + near - coef * (far - near)) - (2.0 * near) / (far + near);
```

Comment:
Each time I see a modification to the OpenGL offset of polylines, the modification seems random. What is the rational for the modification?

Reply:
the result of that formula should be 0 with that coef, but it isn't for some reason. I replaced it with a real 0 and it fixes the bug. There shouldn't be any offset there, from an earlier decision.